### DISCUSSION TITLE
Werewolves - Become a Beast!

### DISCUSSION THREAD

**Discussion by goldrichman9Modded** [DynamicTreesTeam/DynamicTrees#891] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/werewolves-become-a-beast
+ 1 new biome

**Comment by goldrichman9Modded** [comment]
Is this being worked on?

**Reply by Groupix05** [reply]
Yes it's done, we now need to wait for Max to publish it on curseforge in March

**Reply by goldrichman9Modded** [reply]
Alr can’t wait

**Reply by Groupix05** [reply]
It's now on curseforge https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-vampirism

**Reply by goldrichman9Modded** [reply]
> It's now on curseforge https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-vampirism

Alr thxs cant wait to try the mod, also cant wait for 1.21.1

**Reply by goldrichman9Modded** [reply]
> It's now on curseforge https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-vampirism

You  guys should also try uploading in modirth

**Comment by TrueRath** [comment]
To help add info to this post:

[CurseForge](https://www.curseforge.com/minecraft/mc-mods/werewolves-become-a-beast)
[Modrinth](https://modrinth.com/mod/werewolves)

I personally am hoping for 1.20.1. However the mod also has 1.20.4, 1.21, & 1.21.1

Biomes (1):

- Werewolf Heaven


Trees (2):
- Jacaranda Trees
- Magic Trees

Note: I'm not sure if this is important, but Spruce trees also spawn in the Werewolf Heaven biome.

**Comment by supermassimo** [comment]
This has now been added to the vampirism addon https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-vampirism
